README.md

```diff
@@ -339,18 +339,3 @@ Here is an example of a document which can be read/edited in mongo-express (medi
       "func": Code(function() { alert('Hello World!') }),
       "symbol": Symbol("test")
     }
-
```

Comment:
Probably good to add a link reference to the file, so easily the people can check it

Reply:
I think no (I checked some famous projects). License can be viewed in About section of Github.